web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.js

```diff
@@ -56,10 +55,10 @@ odoo.define('web.web_ir_actions_act_window_message', function (require) {
             {
                 return {
                     text: button_definition.name || 'No name set',
-                    oe_link_class: button_definition.oe_link_class ||
+                    classes: button_definition.oe_link_class ||
```

Comment:
@zakiuu  Yes indeed, but I think you commented on an outdated version. Anyway, I used the OCA migration workflow, everything should be ok now.